.github/workflows/integrationTest.yml

```diff
@@ -524,36 +516,6 @@ jobs:
           retry_wait_seconds: 5
           command: cd cd integration/terraform/ec2/win && terraform destroy --auto-approve -var="ami=${{ matrix.arrays.ami }}"
 
-  StopLocalStack:
```

Comment:
What is this change for?

Reply:
The `PerformanceTrackingTest` is also using `localstack` to perform in GovCloud (not sure why). So `StopLocalStack` should have the `PerformanceTrackingTest` as a precondition before executing it. Therefore, I move the `StopLocalStack` to the end to wrap the `PerformanceTrackingTest` too.

Reply:
There is no reference to govcloud

Reply:
Let's me changing the wording a little bit. Our EC2 Linux terraform script uses `localstack`; so Performance Test also uses them. Therefore, there are two changes:
* Added the `PerformanceTrackingTest` as a precondition before executing `StopLocalStack`
* Moving the `StopLocalStack` to wrap around `PerformanceTrackingTest`

Reply:
How does this relate to changing the sha date variable?

Reply:
Not much relates to the PR but its a small thing so I thought I can introduce it also in this PR too.

Reply:
This is a majority of the changes introduced in this PR. "small" is a relative term. We shouldn't need localstack for the performance tests and we can look at that separately. Please revert this change out of the PR unless it directly relates to the sha date to fix our CI.